conda_forge_tick/mamba_solver.py

```diff
@@ -228,6 +247,45 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         self.write()
 
 
+def _get_run_exports(link_tuple):
+    c, pkg, jdata = link_tuple
+
+    with tempfile.TemporaryDirectory() as tmpdir:
+        try:
```

Comment:
would it be possible to make this try except smaller and stuff some of it into an else? If `json.load` fails we might not want to catch that.

Reply:
Right. I agree this could be smaller. What error do we want to throw here? Should we throw an error for anything?

Reply:
Actually, thinking through this further, I think we should keep the code as-is. Basically, if we cannot find a run export, I'd rather us issue an erroring PR than not issue a PR at all. I'm going to merge. If we find this is swallowing a lot of errors we'd rather not, we can address it.